funnel/templates/proposal.html.jinja2

```diff
@@ -238,8 +238,6 @@
           <div class="mui--hide">
             <input type="hidden" name="form.id" value="newcomment"/>
             {{ commentform.hidden_tag() }}
-            {{ commentform.parent_id() }}
-            {{ commentform.comment_edit_id() }}
```

Comment:
Were these redundant? `hidden_tag` is supposed to include all hidden fields, so if we were repeating these fields, that's a good catch.

Reply:
Yeah. These were redundant. `hidden_tag` already had these.